tests/hpc/slurm_db.pm

```diff
@@ -15,6 +15,15 @@ use lockapi;
 use utils;
 use version_utils 'is_sle';
 
+our $slurm_pkg;
+BEGIN {
+    if (get_var('SLURM_VERSION') eq 'slurm') {
+        $slurm_pkg = 'slurm';
+    } else {
+        $slurm_pkg = 'slurm_' . get_var('SLURM_VERSION');
+    }
+}
+
```

Comment:
You duplicate same lines of code across different files. It should be function in utils.

Alos can be done in more simple way:
```
my $slurm_version;
my $slurm_pkg  = $slurm_version ? 'slurm_' . $slurm_version : 'slurm';
```

Reply:
This block will run during early on in the compilation. As such i avoid to invoke another subroutine in runtime. i though that it will make it a bit faster and clever. 
Now i am not sure what to do with your suggestion

Reply:
i just notice that the CI breaks because of this, and i see why pcervinka asked for a lib function. need to look on that further

Reply:
Just put it into hpc utils. There is no reason (bit faster and clever argument looks strange in this context) to duplicate line across files.

Reply:
Please ask me. because i feel we are talking about different things. 
In any case, changes have been made and it is ready for review